lib/tasks/clinics.rake

```diff
@@ -33,7 +33,7 @@ namespace :clinics do
 
     location =
       Location.create!(
-        type: :clinic,
+        type: :community_clinic,
```

Comment:
does this change mean we're missing some test coverage? prob worth adding as this is an important bit of ops.

Reply:
I think at the moment we have no tests for our Rake tasks, which I agree is probably would adding.